src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -121,6 +121,38 @@ class FieldBlocks extends React.Component<FieldTypeProps<Array<BlockEntry>>> {
         return addButtonText;
     }
 
+    @computed get collapsable() {
+        const {
+            schemaOptions: {
+                collapsable: {
+                    value: collapsable,
+                } = {},
+            },
+        } = this.props;
+
+        if (collapsable !== undefined && typeof collapsable !== 'boolean') {
+            throw new Error('The "block" field types only accepts booleans as "collapsable" schema option!');
+        }
+
+        return collapsable;
+    }
+
+    @computed get movable() {
```

Comment:
we need to document this

Reply:
Done in https://github.com/sulu/sulu-docs/pull/614